### PR TITLE
New Workflow: Top Remediations from Teams

### PR DESCRIPTION
**Pull request by sengleson-r7**

### CODE REVIEW

File: workflows/Top_Remediations_For_Asset_In_Teams/help.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+# Version History
+
+* 1.0.0 - Initial workflow
+
+
+# Links
+
+## References
+
+* [Slack](https://slack.com)
```
Comment (zyoutz-r7) [inline workflows/Top_Remediations_For_Asset_In_Teams/help.md:62]:
```suggestion
* [Microsoft Teams](https://teams.microsoft.com)
* [Microsoft Teams Setup](https://insightconnect.help.rapid7.com/docs/microsoft-teams)
```

File: workflows/Top_Remediations_For_Asset_In_Teams/Top_Remediations_For_Asset_In_Teams.icon [hunk 2]
```diff
@@ -0,0 +1,2818 @@
...
+              "slugName": "type_converter",
+              "slugVersion": "1.5.1",
+              "imageData": "https://us.cdn-assets.connect.insight.rapid7.com/icons/rapid7/type_converter/1.5.1/icon.png"
+            },
+            "identifier": "string_to_object",
+            "continueOnFailure": false,
+            "isDisabled": false,
+            "parameters": {
+              "input": {
+                "input": "{\"team\":\"Your Team\",\"channel\":\"Your Channel\"}"
```
Comment (zyoutz-r7) [inline workflows/Top_Remediations_For_Asset_In_Teams/Top_Remediations_For_Asset_In_Teams.icon:981]:
```suggestion
                "input": "{\"team\":\"changeme\",\"channel\":\"changeme\"}"
```
Are we using `changeme` here?

Comment (jschipp-r7) [reply workflows/Top_Remediations_For_Asset_In_Teams/Top_Remediations_For_Asset_In_Teams.icon:981]: Should be `change_me` per the convention document

File: workflows/Top_Remediations_For_Asset_In_Teams/help.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+* InsightConnect License
+* [Microsoft Teams](https://insightconnect.help.rapid7.com/docs/microsoft-teams)
+
+# Documentation
+
+## Setup
+
+Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
+
+Once the workflow has been imported, **edit the Settings step to specify the Microsoft Teams team name and channel name to suit your Teams environment!** Edit the input with the preset text of `Your Team` and `Your Channel` in each Teams step in the workflow.
```
Comment (jschipp-r7) [inline workflows/Top_Remediations_For_Asset_In_Teams/help.md:23]: Should be using `change_me` from the convention doc

File: workflows/Top_Remediations_For_Asset_In_Teams/help.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+After configuring the Settings step with your Microsoft Teams details, activate the workflow in order to trigger it.
+
+### Usage
+
+*This workflow will only trigger in the channel specified in the Microsoft Teams workflow steps.*
+
+To run the workflow, send a message to the specified Microsoft Teams channel starting with the command `!insert-command-here`.
```
Comment (jschipp-r7) [inline workflows/Top_Remediations_For_Asset_In_Teams/help.md:31]: Shouldn't we be less generic here and just show the command this workflow was designed for?

File: workflows/Top_Remediations_For_Asset_In_Teams/help.md (hunk 3 above)
Comment (sengleson-r7) [reply workflows/Top_Remediations_For_Asset_In_Teams/help.md]: fixed

File: workflows/Top_Remediations_For_Asset_In_Teams/help.md (hunk 4 above)
Comment (sengleson-r7) [reply workflows/Top_Remediations_For_Asset_In_Teams/help.md]: yep, good catch - fixed

**Pull request merged**